utilities/resourceRemoval/Remove-vWan.ps1

```diff
@@ -0,0 +1,82 @@
+function Remove-Resource {
+    param(
+        $resourcesToRemove
+    )
+    $resourcesToRetry = @()
+    Write-Verbose '----------------------------------' -Verbose
+    foreach ($resource in $resourcesToRemove) {
+        Write-Verbose ('Trying to remove resource [{0}] of type [{1}] from resource group [{2}]' -f $resource.Name, $resource.ResourceType, $resource.ResourceGroupName) -Verbose
+        try {
+            $null = Remove-AzResource -ResourceId $resource.ResourceId -Force -ErrorAction Stop
+        } catch {
+            Write-Warning ('Removal moved back for re-try. Reason: [{0}]' -f $_.Exception.Message)
+            $resourcesToRetry += $resource
+        }
+    }
+    Write-Verbose '----------------------------------' -Verbose
+    return $resourcesToRetry
+}
+
+function Remove-vWan {
+
+    [Cmdletbinding(SupportsShouldProcess)]
+    param(
+        [Parameter(Mandatory = $false)]
+        [string] $moduleName = 'virtualWans',
+
+        [Parameter(Mandatory = $false)]
+        [string] $ResourceGroupName = 'validation-rg'
+    )
+
+    #Identifying resources
+    $retryCount = 1
+    $retryLimit = 40
+    $retryWaitInSeconds = 15
```

Comment:
Suggestion: Put these as parameters with default values.
Wont do it in the suggestion, but need to also fix casing on params and documentation on functions.

```suggestion
    [Cmdletbinding(SupportsShouldProcess)]
    param(
        [Parameter(Mandatory = $false)]
        [string] $moduleName = 'virtualWans',

        [Parameter(Mandatory = $false)]
        [string] $ResourceGroupName = 'validation-rg',
        
        [Parameter(Mandatory = $false)]
        [int] $retryLimit = 40,
        
        [Parameter(Mandatory = $false)]
        [int] $retryWaitInSeconds = 15
    )

    #Identifying resources
    $retryCount = 1
```

Reply:
done